### PR TITLE
feat: add semantic release

### PR DESCRIPTION
**Pull request by Graborg**
**Ticket:** [#84](https://trello.com/c/dOywK9BF/84-add-semantic-release)
**Intent:** Add semantic releases
**How to test (optional):** Hard to test this one. It is possible to run with --dry-run and --no-ci.
Otherwise we need to merge it to see if it works in travis for real

### CODE REVIEW

**Comment by lessp** [comment]
Sweet! Does it skip publishing to NPM?

File: .travis.yml [hunk 1]
```diff
@@ -15,26 +15,25 @@ jobs:
...
-        - openssl aes-256-cbc -K $encrypted_6b48726482ce_key -iv $encrypted_6b48726482ce_iv
-          -in ./infrastructure/travis/openshift-token.enc -out ./infrastructure/travis/openshift-token -d
       install:
         - bash ./infrastructure/travis/install-openshift.sh
         - export PATH=$PATH:/tmp/openshift
-      script: bash ./infrastructure/travis/deploy.sh
+        - npm i -D @semantic-release/commit-analyzer @semantic-release/release-notes-generator
+          @semantic-release/github @semantic-release/exec semantic-release-docker
+      script:
+        - docker build -f Dockerfile -t jobtechswe/myskills-api .
```
Comment (lundeeell) [inline .travis.yml]: myskills-api? :O

Comment (lundeeell) [reply .travis.yml]: Remove build.sh and add build args from that file here.

**Comment by Graborg** [comment]
> Sweet! Does it skip publishing to NPM?

Yes, it skips it :)

**Pull request merged**